src/renderer/windows/main/components/Snippets.vue

```diff
@@ -14,29 +14,33 @@
                     v-model="searchSnippets"
                     class="rounded flex-1 py-2 px-4"
                     :class="['bg-black-light text-grey-light border border-black-darkest', 'border text-grey-darkest'][theme]"
+                    tabindex="3"
                 >
                 <!-- <span class="ml-4 font-bold w-6 flex items-center justify-center cursor-pointer">A</span> -->
                 <!-- <span class="ml-4 font-bold w-6 flex items-center justify-center cursor-pointer">↓</span> -->
             </div>
             <div class="mb-8 mt-8 overflow-y-scroll overflow-x-visible flex-1 custom-width-for-shadows px-4 -ml-4" ref="list">
                 <div
-                    class="items-center h-12 flex pt-4 pb-3 px-6 mb-4 rounded cursor-pointer clickable border-b-4 border-transparent"
+                    class="w-full items-center justify-between h-12 flex pt-4 pb-3 px-6 mb-4 rounded cursor-pointer clickable border-b-4 border-transparent"
                     :class="{
                         'bg-grey-darkest shadow-md': theme === 0,
                         'bg-grey-light': theme === 1,
                         'border-white': editing && editing.id === snippet.id && theme === 0,
                         'border-brand-blue': editing && editing.id === snippet.id && theme === 1,
                     }"
+                    role="button"
+                    tabindex="0"
                     v-for="snippet in filteredSnippets"
                     :key="snippet.id"
+                    @keyup.enter="edit(snippet)"
                     @click="edit(snippet)"
                 >
                     <span class="flex-1">{{ snippet.key }}</span>
                     <span class="px-3 ml-2 text-grey-darkest py-1 text-xs bg-grey rounded-full" v-if="snippet.regex">regex</span>
                     <span class="px-3 ml-2 text-grey-darkest py-1 text-xs bg-grey rounded-full" v-if="snippet.type === 'js'">js</span>
-                    <span class="flex items-center ml-2" :class="['', 'text-grey-darkest'][theme]" @click.stop="remove(snippet)">
+                    <button tabindex="0" class="flex items-center ml-2 " :class="['text-grey-light', 'text-grey-darkest'][theme]" @click.stop="remove(snippet)">
```

Comment:
It should have a `type="button"` attribute so there is no unexpected behaviour with accidental form submissions.